invoicing/models.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from enumfields import EnumField
+
+from invoicing.enums import (
+    InstallmentPercentageSpecifier,
+    InstallmentType,
+    InstallmentUnit,
+)
+
+
+class InstallmentBase(models.Model):
+    type = EnumField(InstallmentType, verbose_name=_("type"), max_length=32)
+    value = models.DecimalField(
+        verbose_name=_("value"), max_digits=16, decimal_places=2
+    )
+    unit = EnumField(InstallmentUnit, verbose_name=_("unit"), max_length=32)
+    percentage_specifier = EnumField(
+        InstallmentPercentageSpecifier,
+        verbose_name=_("percentage specifier"),
+        max_length=32,
+        blank=True,
+        null=True,
+    )
+    account_number = models.CharField(max_length=255, verbose_name=_("account number"))
+    due_date = models.DateField(verbose_name=_("due date"), blank=True, null=True)
```

Comment:
This is some reason in a plural format in ResKo screenshot. If this is ok, then this comment can skip.

Reply:
Mikko confirmed that this is ok.